PYTHIA6/AliPythia6/AliGenPythia.cxx

```diff
@@ -54,9 +54,9 @@ ClassImp(AliGenPythia)
 
 AliGenPythia::AliGenPythia():
     AliGenMC(),
-    fProcess(kPyCharm),          
+    fProcess(kPyCharm),
```

Comment:
In this class, there are many diff lines like this, where no real intervention happened. It would be good to avoid this, although I do not know how to redo the commit

Reply:
I have tried to recover the master version re-adding my modification but nothing has chenged...